pyproject.toml

```diff
@@ -46,7 +46,7 @@ dependencies = [
     "python-dateutil>=2.8",
     "rapidfuzz>=1.8",
     "requests>=2.26",
-    "sqlalchemy>=1.4,<2",
+    "sqlalchemy>=1.4,<3",
```

Comment:
Will it still work with `sqlalchemy<2` ?

Reply:
The tests are still passing for me locally with sqlalchemy 1.4.46. That might be helpful to add to the test matrix if we want to continue supporting <2 moving forward.

Reply:
My understanding is that `sqlalchemy==1.4` supports all the big changes for `sqlalchemy==2`. Only the `future=True` flags might change behavior (like [this](https://docs.sqlalchemy.org/en/14/changelog/migration_14.html#cascade-backrefs-behavior-deprecated-for-removal-in-2-0)).